arch/arm/core/cortex_m/mpu/arm_mpu.c

```diff
@@ -354,6 +359,21 @@ static int arm_mpu_init(struct device *arg)
 	}
 #endif
 
+#ifdef CONFIG_COVERAGE_HW_TEST
```

Comment:
will this only work on MPU systems?

Reply:
it can work on non mpu systems aswell. But special code it needed for userspace.